stage-0-proposals.md

```diff
@@ -53,3 +59,13 @@ See also the [active proposals](README.md), [stage 1 proposals](stage-1-proposal
 [async-context]: https://github.com/legendecas/proposal-async-context
 [async-context-notes]: https://github.com/tc39/notes/blob/HEAD/meetings/2020-07/july-23.md#async-context-updates--for-stage-1
 [string-trim-characters]: https://github.com/Kingwl/proposal-string-trim-characters
+[array-async-from]: https://github.com/js-choi/proposal-array-async-from
+[explicit-this-param]: https://github.com/gilbert/es-explicit-this
+[explicit-this-param-notes]: https://github.com/tc39/notes/blob/master/meetings/2020-02/february-6.md#syntax-for-explicitly-this-argument-for-stage-1
+[this-arg-reflection]: https://github.com/hax/proposal-function-this
+[this-arg-reflection-notes]: https://github.com/tc39/notes/blob/master/meetings/2020-02/february-6.md#function-thisargumentexpected-property
+[index-from-end]: https://github.com/hax/proposal-index-from-end
+[index-from-end-notes]: https://github.com/tc39/notes/blob/master/meetings/2021-01/jan-28.md#index-from-end-syntax
+[json-slashes-hint]: https://gist.github.com/hax/5691ca8acdf9179e63043857cdc3616b
+[serializer]: https://github.com/Jack-Works/proposal-serializer
```

Comment:
this has never been placed on an agenda - @Jack-Works, is this something you plan to present?

Reply:
Not yet, because I didn't found a proper way to design this

Reply:
in that case, @pd4d10, please remove this one; @Jack-Works can PR it in when they're ready to present it.

Reply:
I'm not sure presentation plan is the hard requirement. There are some proposals in the table never be presented and obviously have no presentation plan (actually I think some of them are more close to inactive, and it's a separate issue).

One important benefit to list a proposal which already have champion (even the champion may not ready to present) is to increase the visibility of the proposals and could help to collect the feedback.

Reply:
There is no hard requirement; as far as I'm aware every single proposal in the table was presented at some point, although some were discussed prior to ES6.

There's not much benefit to collecting feedback from this list on something that hasn't even been presented to the committee - if TC39 doesn't want to spend further time on it, then all feedback is irrelevant. That's what the discourse is for.

Reply:
Adding the visibility can help future people to avoid duplicated work I guess

Reply:
Perhaps, but if it's never been presented, I'm not sure it matters - and again, that's what the discourse is for.

Reply:
Collecting feedback is important to the author/champion of the proposals. For example, Jack may meet some problem on some issues of the proposal, but feedback might help to solve the problem. 

It's vague to predict whether TC39 want to "spend further time" on anything before presenting or even after presenting, actually many stage 1 proposal (and also the stage 0 proposals already in the page) also have serious issues which seems hard to advance. So IMO the only important criteria is whether it have a champion. 

And I don't think discourse is a good place to do continuous work, to me discourse is just a place to gather the ideas and it's very hard to keep watching it.

Reply:
There has to be some filter, or else the stage zero list becomes too long to be useful and maintainable. The criteria we’ve always applied, and I’ll continue to apply, is “presented or on an agenda”.

Reply:
It's definitely useful. Many times I remember I have seen someone wrote the proposal but can't find it, not mention those who never know them. And if we only use a simple criteria (whether it have a champion) there will be no much maintaining work. We could also add action in agenda repo so when new proposal was put in the agenda it could automatically mark it as "ready to present for stage 1".

Reply:
We have a simple criteria that helps filter out all the unpresented ideas that champions have.